blue_block1_x: 0.058
blue_block1_y: -0.2083
blue_block2_x: -0.058
blue_block2_y: -0.2083
red_block1_x: -0.058
red_block1_y: -0.092
red_block2_x: 0.058
red_block2_y: -0.092
